src/picongpu/include/plugins/radiation/Radiation.kernel

```diff
@@ -221,11 +221,15 @@ void kernelRadiationParticles(ParBox pb,
          * inside the superCell (anymore)
          */
         while (isValid)
-          {
+        {
             // only threads with particles are running
             if (linearThreadIdx < particlesInFrame)
-              {
+            {
 
+                PMACC_AUTO(par,(*frame)[linearThreadIdx]);
+                // get old and new particle momenta
+                const vector_32 particle_momentumNow = vector_32(par[momentum_]);
```

Comment:
This is not changed, it was only moved up in the code: see https://github.com/ComputationalRadiationPhysics/picongpu/pull/996/files#diff-05b7cb48a589bb79b6709c1bbc94dcd6L279